Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
 //Write code for the 1st task below this
-
-
+function addTWo(num1,num2){
+    return num1+num2;
+}
+console.log(addTWo(3,7))
 //Write your code above this line
```

Comment:
The console.log statement is missing a semicolon at the end, it should be added to follow best practices.